applications/nrf_desktop/bootloader_dfu.rst

```diff
@@ -200,9 +200,14 @@ You can enhance security further by enabling the following sysbuild Kconfig opti
 * ``SB_CONFIG_BOOT_SIGNATURE_TYPE_PURE`` - This option enables using a pure signature of the image, verifying signature directly on image, rather than on its hash.
   However, you cannot use this option if the secondary image slot uses external memory.
 * ``SB_CONFIG_MCUBOOT_SIGNATURE_USING_KMU`` - This option enables using Key Management Unit (KMU) to store keys for signature verification instead of compiling key data into the MCUboot bootloader image.
-  Using KMU requires provisioning the public key manually.
+  To use KMU, the public key must first be provisioned.
   See the :ref:`ug_nrf54l_developing_provision_kmu` documentation for details.
 
+..note:
+
+    To use automatic provisioning, enable the ``SB_CONFIG_MCUBOOT_GENERATE_DEFAULT_KMU_KEYFILE`` Kconfig option.
```

Comment:
```suggestion
    To use automatic provisioning, enable the :kconfig:option:`SB_CONFIG_MCUBOOT_GENERATE_DEFAULT_KMU_KEYFILE` sysbuild Kconfig option.
```

Reply:
it won't be searchable because only non-sysbuild Kconfigs are shown in the search, that's a different issue, however it highlights that it is a Kconfig option

Reply:
@nordicjm : All the sysbuild configurations are represented in double backticks, unlike the other normal Kconfig options, because these options are not in the Kconfig search and hence don't link. Please see some examples here: 

* https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev/config_and_build/sysbuild/zephyr_samples_sysbuild.html
* https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/migration/migration_sysbuild.html#network_core

The earlier representation was correct.

Reply:
That is used for sysbuild Kconfigs too in zephyr, see https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/releases/release-notes-4.2.rst?plain=1#L261

Reply:
Thank you @nordicjm. This was quick. The sysbuild Kconfig options now link as well. Tested a few doc pages and works fine. We will fix the syntax to ":kconfig:option:`SB" for other sysbuild config options in a separate PR. We already have a Jira open by Jan for nRF desktop.